src/libraries/System.Private.CoreLib/src/System/Numerics/Matrix4x4.cs

```diff
@@ -224,7 +224,7 @@ public Vector3 Translation
         public static Matrix4x4 Add(Matrix4x4 value1, Matrix4x4 value2)
             => (value1.AsImpl() + value2.AsImpl()).AsM4x4();
 
-        /// <summary>Creates a spherical billboard that rotates around a specified object position.</summary>
+        /// <summary>Creates a right-handed spherical billboard matrix that rotates around a specified object position.</summary>
```

Comment:
Once this is merged, please also follow up with a PR to update [Matrix4x4.xml (dotnet/dotnet-api-docs)](https://github.com/dotnet/dotnet-api-docs/blob/main/xml/System.Numerics/Matrix4x4.xml#L289). Once seeded from the original comments, we don't automatically flow changes over to the API docs repo.